api/admin/opds.py

```diff
@@ -19,16 +19,11 @@ def annotate_work_entry(self, work, active_license_pool, edition, identifier, fe
 
         super(AdminAnnotator, self).annotate_work_entry(work, active_license_pool, edition, identifier, feed, entry)
 
-        if isinstance(work, BaseMaterializedWork):
-            data_source_name = work.name
-        else:
-            data_source_name = active_license_pool.data_source.name
-
         feed.add_link_to_entry(
             entry,
             rel="http://librarysimplified.org/terms/rel/refresh",
             href=self.url_for(
-                "refresh", data_source=data_source_name,
```

Comment:
These were adding an unnecessary query parameter now that data source is no longer part of these routes.